cmd/clusterctl/cmd/config_providers.go

```diff
@@ -105,6 +105,48 @@ func runGetRepositories() error {
 	return nil
 }
 
-func runGetRepository(providerName, targetNamespace, watchingNamespace, output string) error {
+func runGetComponents(providerName, targetNamespace, watchingNamespace, output string) error {
+	c, err := client.New(cfgFile)
+	if err != nil {
+		return err
+	}
+
+	components, err := c.GetProviderComponents(providerName, targetNamespace, watchingNamespace)
+	if err != nil {
+		return err
+	}
+
+	if output == "yaml" {
+		return componentsYAMLOutput(components)
+	}
+	return componentsDefaultOutput(components)
+}
+
+func componentsYAMLOutput(c client.Components) error {
+	yaml, err := c.Yaml()
+	if err != nil {
+		return err
+	}
+
+	_, err = fmt.Fprintln(os.Stdout, string(yaml))
```

Comment:
Why Fprintln to stdout instead of just Println?

Reply:
fixed